help/md/dnBirthDeathRateShifts

```diff
@@ -0,0 +1,24 @@
+## name
+dnBirthDeathRateShifts
+## title
+## description
+The birth death with rate shifts (BDS) model describes the distribution of a matrix of species origination and extinction times under complete species sampling and incomplete fossil sampling.
+## details
+Fossil species are represented by a collection of fossil occurrences with uncertainty. Speciation, extinction and sampling rates may be time-homogeneous or piecewise time-heterogeneous. If time-heterogeneous rates are provided, then a vector of rate change time-points musts also be provided. If only a subset of fossil occurrences is provided for each species (e.g. only first and last occurrencces), then the remaining number of fossil samples may be marginalized by specifying `complete=FALSE`.
+## authors
+Walker Pett
+## see_also
+dnFossilizedBirthDeathRange
+## example
+lambda ~ dnExp(10)
+mu ~ dnExp(10)
+psi ~ dnExp(10)
+
+bd ~ dnBDS(lambda=lambda, mu=mu, psi=psi, rho=1, taxa=taxa, complete=FALSE)
```

Comment:
How about instead of having a separate distribution, the PyRate "BDS" model is implemented as a special case of the FBDRMatrix model where the we assume there are no unsampled lineages? In other words, this model is specified by setting `bds=true` in `FBDRMatrix` or something similar.

Reply:
For now, though, as far as I can tell there is no other distribution named similarly to `BDS` or `BirthDeathRateShifts`, so this naming should be safe.